styles.css

```diff
@@ -42,17 +45,8 @@ h1 {
   text-decoration: none;
   display: inline-block;
   border-radius: 25px;
-  font-size: 2rem;
+  font-size: 1.25rem;
   cursor: pointer;
   box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
 
 }
-
```

Comment:
I like that you were able to do this without needing to use any media queries. This is also my first code comment on git hub